sim.c

```diff
@@ -294,10 +296,11 @@ static void apply_conduit(struct solution *solution, struct board *board, struct
                     // bonds are consumed even if the atoms aren't.
                     *b &= ~(ALL_BONDS & ~RECENT_BONDS & a);
                 }
-            } else {
-                struct vector p = mechanism_relative_position(other_side, delta.u, delta.v, 1);
+                p = mechanism_relative_position(other_side, delta.u, delta.v, 1);
                 rotate_bonds(&a, rotation);
-                insert_atom(board, p, a, "conduit output");
+                insert_atom(board, p, a | BEING_PRODUCED, "conduit output");
+            } else {
+                *lookup_atom(board, p) &= ~BEING_PRODUCED;
```

Comment:
This is just an annoying code style comment that single-line elses shouldn't have braces (feel free to disregard but I thought I'd mention it anyway)